l10n_cn_fapiao/data/fapiao.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="fapiao_tax_type_13" model="fapiao.tax.type">
```

Comment:
add the 6% type as well which is quite common
Add a "active" field to the model